herddb-core/src/test/java/herddb/sql/RawSQLTest.java

```diff
@@ -2738,6 +2739,25 @@ public void createIndexOnTableTest() throws Exception {
         }
     }
 
+    @Test
+    public void openJPATest() throws Exception {
+        String nodeId = "localhost";
+        try (DBManager manager = new DBManager("localhost", new MemoryMetadataStorageManager(), new MemoryDataStorageManager(), new MemoryCommitLogManager(), null, null)) {
+            manager.start();
+            assertTrue(manager.waitForBootOfLocalTablespaces(10000));
+            execute(manager, "CREATE TABLE `herd`.`parentTable` (`k1` VARCHAR NOT NULL, `p1` INTEGER)", Collections.emptyList());
```

Comment:
Do we have a test on backtick for tablespace with DML?

Reply:
yep, we have a few tests about backticks, but as we can see here we did not have them for "ALTER TABLE"

btw OpenJPA Dictionary will delimit every identifier, this is done because of the aggressive list of reserved words on Apache Calcite